frontend/src/components/common/LoginBtn/LoginModal/LoginModal.jsx

```diff
@@ -0,0 +1,231 @@
+              </p>
+            </div>
+            <div className={styles.closeBtn}>
+              <IconButton icon={<ICONS.closeMobile />} onClick={toggleLogin} />
+            </div>
+          </div>
+
+          <div className={styles.formContainer}>
+            <form onSubmit={handleSubmit}>
+              <div className={styles.dataContainer}>
+                <div className={styles.emailContainer}>
+                  <label className={styles.labelData} htmlFor="email">
+                    Ел. пошта {submitted && !email && <span>*</span>}
+                  </label>
+
+                  <input
+                    id="email"
+                    className={styles.dataInput}
+                    type="email"
+                    pattern="^[a-zA-Z0-9._-]+@[a-zAZ.-]+\.[a-zA-Z]{2,4}$"
+                    placeholder="Введіть електронну пошту"
+                    value={email}
+                    onChange={(e) => {
+                      setEmail(e.target.value.trim());
+                      setEmailError('');
+                    }}
+                  />
+                  {submitted && emailError && (
+                    <p className={styles.errorMessage}>{emailError}</p>
+                  )}
+                  {submitted && !email && (
+                    <p className={styles.errorMessage}>
+                      Введіть електронну пошту
+                    </p>
+                  )}
+                  {submitted && email && !validateEmail(email) && (
+                    <p className={styles.errorMessage}>
+                      Невірний формат електронної пошти
+                    </p>
+                  )}
+                  {resetPassword && !email && (
+                    <p className={styles.errorMessage}>
+                      Введіть електронну пошту
+                    </p>
+                  )}
+                  {resetPassword && email && !validateEmail(email) && (
+                    <p className={styles.errorMessage}>
+                      Невірний формат електронної пошти
+                    </p>
+                  )}
+                  {submitted && resetEmailError && (
+                    <p className={styles.errorMessage}>{resetEmailError}</p>
+                  )}
+                </div>
+
+                <div className={styles.passwordContainer}>
+                  <label className={styles.labelData} htmlFor="password">
+                    Пароль {submitted && !password && <span>*</span>}
+                  </label>
+
+                  <input
+                    id="password"
+                    name="password"
+                    className={styles.dataInput}
+                    type="password"
+                    placeholder="Введіть ваш пароль"
+                    value={password}
+                    onChange={(e) => setPassword(e.target.value)}
+                  />
+                  {submitted && !password && (
+                    <p className={styles.errorMessage}>Введіть пароль</p>
```

Comment:
you can check it inside handeSubmit function and then set it to the variable passwordError

Reply:
Done